docs/documentation/platform/access-controls/project-access-requests.mdx

```diff
@@ -34,3 +33,21 @@ To request access to a project you don't currently have access for:
   Project administrators will receive email notification with details regarding
   the access request.
 </Info>
+
+# Gain Access to a Project
+
+This action can be only be performed by Organization Admins or users with the permission `Access all organization projects`.
```

Comment:
Make it a little more clear by saying you must have a role at the organization that has the following permission

Reply:
Updated